jans-cedarling/cedarling/src/jwt/log_entry.rs

```diff
@@ -15,15 +15,16 @@ pub struct JwtLogEntry {
 }
 
 impl JwtLogEntry {
-    /// Shorthand for creating a [`LogType::System`] log.
-    pub fn system(msg: String) -> Self {
-        let base = BaseLogEntry::new_opt_request_id(LogType::System, None);
+    /// Helper for creating a [`LogLevel::WARN`] log.
+    pub fn warn(msg: String) -> Self {
```

Comment:
JwtLogEntry only supports WARN level logs?
To make it flexible, is it better to define a function like following?
`pub fn info(msg: String, level: Option<LogLevel>) -> Self {}`